provisioning_templates/snippet/ansible_provisioning_callback.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+kind: snippet
+name: ansible_provisioning_callback
+-%>
+<% if host_param_true?('ansible_tower_provisioning') -%>
+<%
+  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
+  os_major = @host.operatingsystem.major.to_i
+  has_systemd = (@host.operatingsystem.name == 'Fedora' && os_major >= 20) || (rhel_compatible && os_major >= 7)
+-%>
+<% if has_systemd -%>
+<%= save_to_file('/etc/systemd/system/ansible-callback.service',
+                 snippet('ansible_tower_callback_service')) %>
+# Runs during first boot, removes itself
+/usr/bin/systemctl enable ansible-callback
+<% else -%>
+# Assume systemd is not available
+<%= save_to_file('/root/ansible_provisioning_call.sh', snippet('ansible_tower_callback_script')) %>
+(crontab -u root -l 2>/dev/null; echo "@reboot /root/ansible_provisioning_call.sh" ) | crontab -u root -
```

Comment:
Any reason why this is using an unqualified path where others are using fully paths?

Reply:
Why unqualified? `/root/ansible_provisioning_call.sh` is the full path or am I misunderstanding something?

Reply:
I meant ```crontab``` vs ```/usr/bin/systemctl```

Reply:
I think it's about the `crontab` command